### PR TITLE
chore(graphql-elasticsearch-transformer): add default to lowest instance

### PR DESCRIPTION
**Pull request by akshbhu**
*Issue #3860

*Description of changes:*

* Changed the default instance to t2.micro

* Add the upgrade policy to upragde the version of instance inplace.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js [hunk 1]
```diff
@@ -443,6 +443,44 @@ async function transformGraphQLSchema(context, options) {
 
   if (directiveMap.directives.includes('searchable')) {
     searchableTransformerFlag = true;
+    if (context.input.command === 'add') {
```
Comment (SwaySway) [inline packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js:446]: We could also use`resourcesToBeCreated` here similar to apiProjectIsFromOldVersion.
https://github.com/aws-amplify/amplify-cli/blob/16a2b4f50a85372f733eef129e43b154a993b136/packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js#L219

**Comment by akshbhu** [comment]
Closing as we currently don't support t2.micro instances as they work on lower version 2.3

**Comment by sakhmedbayev** [comment]
> Closing as we currently don't support t2.micro instances as they work on lower version 2.3

@akshbhu, what is the problem with the lower version 2.3?

It would be great if amplify work with t2.micro out of the box. I think a lot of people are disappointed with the high cost of the amplify's default elasticsearch instance, especially at the dev time.

**Comment by Blkc** [comment]
This seems to be related to https://github.com/aws-amplify/amplify-cli/issues/3860#issuecomment-621373818

**Pull request closed without merging**